components/event/EventCard.js

```diff
@@ -57,12 +53,12 @@ export default function EventCard({ manage, event, usernames }) {
           {event.price?.startingFrom > 0 && <TbCoin title="Paid event" />}
           {event.price?.startingFrom === 0 && <TbCoinOff title="Free event" />}
         </div>
-        <div className="flex-1 space-y-1 p-4">
```

Comment:
I do not understand this change?

Reply:
This change is done by my prettier. So, it is just making tailwind css properties arrange alphabetically.

Reply:
We don't have those Prettier settings in the project, please use the Project's settings and style

Reply:
How to find prettier settings that are present for the project?

Reply:
The settings are picked up automatically for me, maybe you have some settings overriding it